src/rgw/rgw_cors.cc

```diff
@@ -40,6 +40,30 @@ void RGWCORSRule::dump_origins() {
   }
 }
 
+void RGWCORSRule::dump(Formatter *f) const
+{
+  f->open_object_section("CORSRule");
+  f->dump_string("ID", id);
+  f->dump_unsigned("MaxAgeSeconds", max_age);
+  f->dump_unsigned("AllowedMethod", allowed_methods);
+  f->open_array_section("AllowedOrigin");
+  for (auto& origin : allowed_origins) {
+    f->dump_string("Origin", origin);
+  }
+  f->close_section();
```

Comment:
does encode_json work for these lists too?

Reply:
Yes, changed